piff/input.py

```diff
@@ -559,6 +487,45 @@ def __init__(self, config, logger=None):
         dec = config.get('dec',None)
         self.setPointing(ra, dec, logger)
 
+    def load_images(self, stars, logger=None):
+        """Load the image data into a list of Stars.
+
+        We don't store the image data for Stars when we write them to a file, since that
+        would take up a lot of space and is usually not desired.  However, we do store the
+        bounds in the original image where the star was cutout, so if you want to load back in
+        the original data from the image file(s), you can do so with this function.
+
+        :param stars:           A list of Star instances.
+        :param logger:          A logger object for logging debug info. [default: None]
+
+        :returns: a new list of Stars with the images information loaded.
+        """
+        logger = galsim.config.LoggerWrapper(logger)
+
+        images = {}
+        weight_images = {}
+        for image_num in range(self.nimages):
```

Comment:
I think loading the images is pretty fast.  There is a numpy array copy, but otherwise not much calculation.  But I'll re-evaluate that once I'm stress testing versions that load tons of objects and pull out a small subset as stars to see if that's a tall pole or not.